### PR TITLE
TypeScript definition

### PR DESCRIPTION
**Pull request by kmxz**
Allows using the package from a TypeScript project

### CODE REVIEW

File: src/wrap.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+declare const wrap: <T>(html: (strings: TemplateStringsArray, ...values: unknown[]) => T) => (strings: TemplateStringsArray, ...values: unknown[]) => T;
+
+export default wrap;
```
Comment (bashmish) [inline src/wrap.d.ts:3]: this is what `tsc` gives me me if I rewrite the JS file in TS with such types, looks a bit shorter and closer to the origin, and my VSCode does not complain about using `const`

```ts
export default function wrap<T>(
  html: (strings: TemplateStringsArray, ...values: unknown[]) => T,
): (strings: TemplateStringsArray, ...values: unknown[]) => T;
```

can you please use the same?

also please format using `yarn format` script, it should and a line in the end and do some other things

**Review by bashmish — CHANGES_REQUESTED** [review]
good idea, thanks!
some small changes are needed, hope you agree and can fix it

File: src/wrap.d.ts (hunk 1 above)
Comment (kmxz) [reply src/wrap.d.ts]: Looks great! Thanks for the suggestion.

**Review by bashmish — APPROVED** [review]

**Comment by bashmish** [comment]
Thanks for your contribution!

**Comment by bashmish** [comment]
Released in https://www.npmjs.com/package/carehtml/v/0.1.2

**Pull request merged**